src/prof.c

```diff
@@ -2744,12 +2744,12 @@ prof_log_stop(tsdn_t *tsdn) {
 	emitter_init(&emitter, emitter_output_json, &prof_emitter_write_cb,
```

Comment:
It looks this does intentionally output to json only?

Reply:
and the two begin functions differ in if `emitter_json_key_prefix` is called; is that required here?

Reply:
`emitter_begin` and `emitter_end` are designed to be called no matter what the output format is.

`emitter_json_object_begin` and `emitter_json_object_end` are designed for printing individual json objects. Despite that the entire json is also an object, from printing point of view there are some subtle differences: `emitter_json_object_begin` assumes that there can be content before it, and `emitter_json_object_end` assumes that there can be content after it, both of which affect the handling of new lines.